data/data/alibabacloud/vpc/slb.tf

```diff
@@ -0,0 +1,138 @@
+
+resource "alicloud_slb_load_balancer" "slb_external" {
+  resource_group_id  = var.resource_group_id
+  load_balancer_name = "${local.prefix}-slb-external"
+  address_type       = "internet"
+  load_balancer_spec = "slb.s2.small"
+
+  tags = merge(
+    {
+      "Name" = "${local.prefix}-slb-external"
+    },
+    var.tags,
+  )
+}
+
+resource "alicloud_slb_listener" "listener_external_80" {
+  load_balancer_id    = alicloud_slb_load_balancer.slb_external.id
+  backend_port        = 80
+  frontend_port       = 80
+  protocol            = "tcp"
+  bandwidth           = 10
+  sticky_session      = "on"
+  sticky_session_type = "insert"
+  cookie_timeout      = 86400
+  # health_check              = "on"
```

Comment:
Any reason for these comments here?

Reply:
We haven't determined some functions here. We will update in a later PR